spek-ide-plugin/interop-jvm/build.gradle

```diff
@@ -1,10 +1,17 @@
 apply from: "$rootDir/gradle/common/dependencies.gradle"
-apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply plugin: 'kotlin-platform-jvm'
+apply plugin: 'com.github.johnrengelman.plugin-shadow'
 
 dependencies {
-    expectedBy project(':spek-ide-plugin:interop-common')
+    compileOnly project(':spek-runtime:jvm')
```

Comment:
hm, why `compileOnly`?

Reply:
I expect the project to provide the dependency for us, hence `compileOnly`.

Reply:
that's what I thought, but then I'm not sure how are we loading project dependencies into Plugin's classpath…?